hack/validate-gomod

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+set -eu
+
+: ${CONTINUOUS_INTEGRATION=}
+: ${DOCKER_BUILDKIT=}
+
+progressFlag=""
+if [ "$CONTINUOUS_INTEGRATION" = "true" ]; then progressFlag="--progress=plain"; fi
+
+cacheOnlyFlag=""
+if ! docker build 2>&1 --help | grep buildx > /dev/null; then cacheOnlyFlag="-o type=cacheonly"; fi
+
+case ${1:-} in
+'')
+  export DOCKER_BUILDKIT=1
+  docker build $progressFlag $cacheOnlyFlag -f ./hack/dockerfiles/gomod.Dockerfile --target validate . || exit 1
+  ;;
+check)
+  status="$(git status --porcelain -- go.mod go.sum 2>/dev/null)"
+  diffs=$(echo "$status" | grep -v '^[RAD] ' || true)
+  if [ "$diffs" ]; then
+      {
+	set +x
+        echo 'The result of "./hack/validate-gomod" differs'
```

Comment:
Looks like you're using spaces for indentation; could you run `shfmt -w .` (using https://github.com/mvdan/sh)?